source/features/download-folder-button.tsx

```diff
@@ -21,15 +21,17 @@ function init(): void {
 		);
 	} else {
 		// "Repository refresh" layout
-		select('.file-navigation > .d-flex:last-child')!.append(
-			<a
-				className="btn ml-2"
-				href={downloadUrl.href}
-			>
-				<DownloadIcon className="mr-1"/>
-				Download
-			</a>
-		);
+		for (const deleteButton of select.all(`form[action^="/${getRepo()!.nameWithOwner}/tree/delete"]:not(.rgh-download-folder)`)) {
+			deleteButton.classList.add('rgh-download-folder');
+			deleteButton.before(
+				<a
+					className="dropdown-item btn-link"
+					href={downloadUrl.href}
+				>
```

Comment:
```suggestion
				<a className="dropdown-item btn-link" href={downloadUrl.href}>
```

Is `btn-link` required here? It's not a `<button>` to begin with

Reply:
I will double check